### PR TITLE
Ports TauCeti's briefcases!

### PR DESCRIPTION
**Pull request by Gandalf2k15**
## About The Pull Request

Ports TauCeti's briefcases: https://github.com/TauCetiStation/TauCetiClassic/pull/8966

image

One less codersprite to worry about.

Adds a good few variants too, also makes the secure ones actually show if they're locked or unlocked.

## Changelog

:cl:
add: New briefcase sprites have been added!
/:cl:

### CODE REVIEW

**Comment by GoldenAlpharex** [comment]
I'd argue the current briefcases aren't codersprites, to be fair.

File: modular_skyrat/modules/more_briefcases/code/briefcases.dm [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	inhand_icon_state = "lockbox"
+
+/obj/item/storage/briefcase/virology
+	name = "virology briefcase"
+	icon_state = "briefcase_vir"
+	inhand_icon_state = "lockbox"
+
+/obj/item/storage/briefcase/engineering
+	name = "engineering briefcase"
+	icon_state = "briefcase_eng"
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/more_briefcases/code/briefcases.dm:26]: Are any of these actually used anywhere though?

**Comment by Gandalf2k15** [comment]
> I'd argue the current briefcases aren't codersprites, to be fair.

image

File: modular_skyrat/modules/more_briefcases/code/briefcases.dm [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+/obj/item/storage/secure/briefcase/update_overlays()
+	. = ..()
+	if(SEND_SIGNAL(src, COMSIG_IS_STORAGE_LOCKED))
+		. += "briefcase_locked"
+	else
+		. += "briefcase_open"
+
+/obj/item/storage/secure/briefcase/white
+	icon_state = "briefcase_secure_black"
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/more_briefcases/code/briefcases.dm]: You forgot to make it the white variant.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Most of these aren't even used anywhere. The engineering briefcase looks transparent, and overall really doesn't look too great.

The inflatable wall is the only sprite out of all of these that I agree is a good change, the other sprites are... I don't know, they just look like they're made out of plastic, I think the current briefcases fit better because they don't look as shiny, they look more like leather than plastic.

File: modular_skyrat/modules/more_briefcases/code/briefcases.dm (hunk 1 above)
Comment (Gandalf2k15) [reply modular_skyrat/modules/more_briefcases/code/briefcases.dm:26]: Not yet, but they will be eventually.

**Comment by Gandalf2k15** [comment]
> Most of these aren't even used anywhere. The engineering briefcase looks transparent, and overall really doesn't look too great.
> 
> The inflatable wall is the only sprite out of all of these that I agree is a good change, the other sprites are... I don't know, they just look like they're made out of plastic, I think the current briefcases fit better because they don't look as shiny, they look more like leather than plastic.

Given our entire aesthetic going with the plastic look (so is TG, see medkits) this change makes them align with those styles.

**Comment by GoldenAlpharex** [comment]
> > I'd argue the current briefcases aren't codersprites, to be fair.
> 
> image

That's not a codersprite though. It's a briefcase with a manual combination lock. I think it looks quite decent, and it's a rather new sprite too, it only got added a few months ago upstream.

**Pull request merged**